sdk/eventhub/event-hubs/samples/typescript/src/iothubConnectionString.ts

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT Licence.
+
+/*
+  This sample demonstrates how to convert an Iot Hubs connection string to
+  an Event Hubs-compatible connection string that can be used with the EventHubConsumerClient
```

Comment:
```suggestion
  an Event Hubs connection string that can be used with the EventHubConsumerClient
```

Reply:
Updated this section based on @ramya-rao-a's feedback.